README.md

```diff
@@ -28,9 +28,10 @@ the result of that.
 
 When done remove the database file and the downloaded content.
 ```
-```
+```shell
 $ gptscript ./example.gpt
-
+```
```

Comment:
I think line 30 needs to be deleted here. This code block starts with two instances of triple backticks.